### PR TITLE
Add some additional information to each package file

### PR DESCRIPTION
**Pull request by castastrophe**
Looking at a few of our published components, they didn't have links back to the GitHub page so I thought I'd add that.  In doing that, I found a few other things to clean up and standardize on as well.  This repo contains edits to package.json files including:

- Additional contributors
- Keywords
- Github URLs

### CODE REVIEW

File: elements/pfe-accordion/package.json [hunk 1]
```diff
@@ -8,25 +8,35 @@
...
   },
-  "author": {
-    "name": "Kyle Buchanan"
-  },
+  "contributors": [
+    {
+      "name": "Kyle Buchanan",
+      "email": "kylebuch8@gmail.com",
+      "url": "https://github.com/kylebuch8"
+    },
```
Comment (starryeyez024) [inline elements/pfe-accordion/package.json]:
```suggestion
    }, {
      "name": "Chris Doherty"
    }, {
      "name": "Kendall Totten"
    },
```

File: elements/pfe-tabs/package.json [hunk 2]
```diff
@@ -8,25 +8,35 @@
...
   },
-  "author": {
-    "name": "Kyle Buchanan"
-  },
+  "contributors": [
+    {
+      "name": "Kyle Buchanan",
+      "email": "kylebuch8@gmail.com",
+      "url": "https://github.com/kylebuch8"
+    },
```
Comment (starryeyez024) [inline elements/pfe-tabs/package.json]:
```suggestion
    }, {
    	  "name": "Chris Doherty",
    }
```

File: elements/pfe-content-set/package.json [hunk 3]
```diff
@@ -6,27 +6,40 @@
...
     "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-content-set/test/"
   },
-  "author": {
-    "name": "cd"
-  },
+  "contributors": [
+    {
+      "name": "cd"
+    },
+    {
```
Comment (starryeyez024) [inline elements/pfe-content-set/package.json]:
```suggestion
    {
          "name": "Chris Doherty"
    }, {
```

File: elements/pfe-accordion/package.json (hunk 1 above)
Comment (castastrophe) [reply elements/pfe-accordion/package.json]: @starryeyez024 I'll get you added too.

Chris had entered himself as `cd` so I wanted to respect that.  Let me know how you want to be cited @chrisdo1.

File: elements/pfe-content-set/package.json (hunk 3 above)
Comment (castastrophe) [reply elements/pfe-content-set/package.json]: See above - Chris listed himself as `cd`

File: elements/pfe-accordion/package.json (hunk 1 above)
Comment (chrisdo1) [reply elements/pfe-accordion/package.json]: @castastrophe @starryeyez024 just cd is fine. Thanks!

**Comment by starryeyez024** [comment]
Lurking Gremlins Tout Mischief 👍  @castastrophe  - just needs master merged in

File: package.json [hunk 4]
```diff
@@ -1,5 +1,17 @@
 {
   "name": "@patternfly/patternfly-elements",
+  "publishConfig": {
```
Comment (mwcz) [inline package.json]: The publishConfig can be removed, it's only needed in the element's package files.  It allows the package to be published to npm, and we don't want to publish the entire monorepo to npm.

File: elements/pfe-accordion/package.json [hunk 5]
```diff
@@ -8,25 +8,38 @@
...
-  "description": "Accordion element for PFElements",
+  "description": "Accordion for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/patternfly/patternfly-elements/tree/v1.0.0-prerelease.10/elements/pfe-accordion"
```
Comment (mwcz) [inline elements/pfe-accordion/package.json]: Per the [npm docs](https://docs.npmjs.com/files/package.json#repository), the repository url field needs to point to a git repo, something you can run `git clone` on, so this URL which points to a subdirectory within a tag doesn't meet that requirement.  Fortunately, the docs offer a solution!  There is an optional `directory` field to specify a subdirectory within the git repo where the given element lives.

```json
"repository": {
  "type" : "git",
  "url" : "https://github.com/patternfly/patternfly-elements.git",
  "directory": "elements/pfe-accordion"
}
```

We can also, optionally, use the `github:patternfly/patternfly-elements` shorthand for the url field.

```json
"repository": {
  "type" : "git",
  "url" : "github:patternfly/patternfly-elements",
  "directory": "elements/pfe-accordion"
}
```

File: elements/pfe-datetime/package.json [hunk 6]
```diff
@@ -11,21 +11,22 @@
...
     "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
     "dev": "../../node_modules/.bin/gulp dev",
     "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-datetime/test/"
   },
-  "author": {
-    "name": "Luke Dary"
-  },
+  "authors": [
+    {
+      "name": "Luke Dary"
```
Comment (mwcz) [inline elements/pfe-datetime/package.json]: @kylebuch8  worked on this as well.

File: elements/pfe-accordion/package.json [hunk 7]
```diff
@@ -8,25 +8,38 @@
...
+    {
+      "name": "Kendall Totten"
+    }
+  ],
   "license": "MIT",
   "dependencies": {
     "@patternfly/pfe-sass": "^1.0.0-prerelease.11",
     "@patternfly/pfelement": "^1.0.0-prerelease.11"
   },
-  "generator-pfelement-version": "0.5.5",
```
Comment (mwcz) [inline elements/pfe-accordion/package.json]: Can we leave the generator version?  It's kind of nice to know what version of the generator was used to generate each element.

File: package.json (hunk 4 above)
Comment (castastrophe) [reply package.json]: +1 fixing that now

File: elements/pfe-accordion/package.json (hunk 5 above)
Comment (castastrophe) [reply elements/pfe-accordion/package.json]: I love that shorthand.

File: elements/pfe-datetime/package.json (hunk 6 above)
Comment (castastrophe) [reply elements/pfe-datetime/package.json]: +1 get credit for your hard work!

File: elements/pfe-accordion/package.json (hunk 7 above)
Comment (castastrophe) [reply elements/pfe-accordion/package.json]: When we last chatted it seemed like you were wavering on this?  Do you want me to add them back?

Comment (castastrophe) [reply elements/pfe-accordion/package.json]: If not, then this is ready for your review

Comment (mwcz) [reply elements/pfe-accordion/package.json]: I decided it's best to keep them so I went ahead and added them back in.  Ready to merge!

**Pull request merged**